sam_bot_description/config/bridge_config.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
I believe we also add in a Camera and Lidar in the Sensors part of the FirstTime setup guide -- but I see in your other PR that its on your checklist :-) Just leaving the note here that its missing

Reply:
Should be good to go with [57db422](https://github.com/ros-navigation/navigation2_tutorials/pull/102/commits/57db4224d143056d692990a95f7cc3ad7a7cb45b).